services/app-web/src/index.tsx

```diff
@@ -132,6 +133,9 @@ if (ldClientId === undefined) {
         </React.StrictMode>
     )
 })().catch((e) => {
+    recordJSException(
```

Comment:
Do have have access to `otel` at this point in the app?

Reply:
nope we don't i was thinking that when I was coming back from PT today. We can't do this lol!